cmds/http-gateway/main.go

```diff
@@ -224,8 +226,17 @@ func main() {
 		logger = logging.WithValuesFromContext(ctx, logging.Logger)
 	)
 
+	if *profServiceName != "" {
+		if err := profiler.Start(profiler.Config{
+			Service:        *profServiceName,
+			ServiceVersion: "v1"}); err != nil {
```

Comment:
let's put this on 2 lines for readability

Reply:
Done

Reply:
I mean doing this:

err := profiler.Start(...)

if err !=nil {

}

A multiline if err := ... ; err != nil is a little cumbersome

Reply:
Done